plugins/ksp/src/org/jetbrains/kotlin/ksp/symbol/impl/java/KSVariableParameterJavaImpl.kt

```diff
@@ -43,6 +43,8 @@ class KSVariableParameterJavaImpl(val psi: PsiParameter) : KSVariableParameter {
         KSTypeReferenceJavaImpl.getCached(psi.type)
     }
 
+    override val hasDefault: Boolean = false
```

Comment:
You can remain this part uncovered by test since I am still working on Java symbol implemented. Replace with a TODO() also works.

Reply:
8ae5910